opensrp-reveal/src/main/java/org/smartregister/reveal/model/TaskDetails.java

```diff
@@ -384,4 +384,12 @@ public void setGroupedTaskCodes(String groupedTaskCodeStatusString) {
         }
         this.groupedTaskCodes = stringBuilder.toString();
     }
+
+    public boolean isNonResidential() {
+        return getFamilyMemberNames() != null && getFamilyMemberNames().equalsIgnoreCase("NR");
```

Comment:
Let's avoid using acronyms. if `NR` means `Non Residential` then use something like `non_residential`. Also move it to a constants file.